code/modules/jobs/job_types/security.dm

```diff
@@ -124,7 +124,7 @@ Detective
 
 	outfit = /datum/outfit/job/detective
 
-	access = list(access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
+	access = list(access_sec_doors, access_security, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
```

Comment:
Take it to the forum and inform the Host(s)/Head Admin(s) if you really think this is the right thing to do.